### PR TITLE
Unified data types

### PR DESCRIPTION
**Pull request by JoshuaAdemola**

### CODE REVIEW

File: content/web-sdk-for-plugins/concepts.md [hunk 1]
```diff
@@ -85,7 +85,7 @@ The following properties are available:
 * **imports**: A list of plugins used by the application. List of `<<applicationName>>/<<pluginName>>`.
 * **noAppSwitcher**: [optional] If set to true, the application will not appear in the app switcher menu. You might want to use this, for example, if the application only exposes plugins.
 * **options**:
-	* **hide_navigator**: [optional] A boolean, which if set to true collapses the navigator menu at the left by default.
```
Comment (BeateRixen) [inline content/web-sdk-for-plugins/concepts.md:88]: I would probably revert this change as this might conflict with the removal of those file in the other PR.

**Comment by BeateRixen** [comment]
@JoshuaAdemola As this is not that much, could you please add Edge, Machine Learning and DataHub?

File: content/web-sdk-for-plugins/concepts.md (hunk 1 above)
Comment (JoshuaAdemola) [reply content/web-sdk-for-plugins/concepts.md]: Fixed

**Comment by JoshuaAdemola** [comment]
> @JoshuaAdemola As this is not that much, could you please add Edge, Machine Learning and DataHub?

@BeateRixen Fixed.

What do I do with "non-boolean"?

**Comment by BeateRixen** [comment]
> 

I would go with "non-Boolean"

**Review by BeateRixen — APPROVED** [review]

**Review by rssag — APPROVED** [review]
